rpm/sailfish-office.spec

```diff
@@ -31,13 +31,24 @@ Summary: Translation source for %{name}
 License: GPLv2
 Group: System/Base
 
+%package tests
+Summary: Unitary tests for %{name}
+License: GPLv2
+Group: System/Base
```

Comment:
Think license gets inherited. Not sure if System/Base is better, could just skip that as well.